### ISSUE TITLE
Support for no colon in UTC offset

### ISSUE THREAD

**Issue by tapatio** [js-joda/js-joda#254]
The ISO 8601 standard allows for the UTC offset to be specified without a colon:

`2018-06-12T19:23:26.880+0000`

Ref: [https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC](url)

However, js-joda only supports UTC offsets with a colon:

`2018-06-12T19:23:26.880+00:00`

Can you please add support for UTC offsets without the colon?

Thanks.

**Comment by pithu** [comment]
Hello @tapatio 

thanks for reporting and for using js-joda.

Actually js-joda default formatter is not parsing full featured ISO but more or less [RFC 3339, section 5.6](https://www.ietf.org/rfc/rfc3339.txt).

Nevertheless you can define your own specific parser pattern with js-joda. Here an example that parses an offset without a colon.    

```
const f = DateTimeFormatter.ofPattern("uuuu-MM-dd'T'HH:mm:ss.SSSX")
ZonedDateTime.parse("2018-06-12T19:23:26.880+0000", f)
````

You can check the doc for the patterns https://js-joda.github.io/js-joda/esdoc/class/src/format/DateTimeFormatter.js~DateTimeFormatter.html and there are a lot of issues about that on stack overflow.

I hope that helps

**Comment by tapatio** [comment]
Thanks for the prompt reply!

I tried the formatter pattern above that uses "X" as defined here:

X      |     zone-offset 'Z' for zero   |    offset-X    | Z; -08; -0830; -08:30; -083015; -08:30:15;

That works and I am able to parse "2018-06-12T19:23:26.880+0000". However, I am then unable to parse "2018-06-12T19:23:26.880+00:00". I tried "x", and "Z" options and they did not work either. 

Is there a way to configure a formatter to parse both "0000" and "00:00"? 

Thanks.
